framework/src/auxkernels/TagVectorAux.C

```diff
@@ -1,35 +0,0 @@
-//* This file is part of the MOOSE framework
-//* https://mooseframework.inl.gov
-//*
-//* All rights reserved, see COPYRIGHT for full restrictions
-//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
-//*
-//* Licensed under LGPL 2.1, please see LICENSE for details
-//* https://www.gnu.org/licenses/lgpl-2.1.html
-
-#include "TagVectorAux.h"
-
-registerMooseObject("MooseApp", TagVectorAux);
-
-InputParameters
-TagVectorAux::validParams()
-{
-  InputParameters params = TagAuxBase<AuxKernel>::validParams();
-  params.addRequiredParam<TagName>("vector_tag", "Tag Name this Aux works on");
```

Comment:
Given the comments about the linear system vectors, I wonder if it's worth keeping this class around and having `TagResidualAux` inherit from it and `params.renameParam("vector_tag", "residual_tag", "new_docstring");`, similarly for `TagSolutionAux`. The new scaling parameter would still go into `TagResidualAux`